src/EditorFeatures/Core.Wpf/Classification/ClassificationTypeFormatDefinitions.cs

```diff
@@ -708,5 +933,22 @@ private XmlLiteralTextFormatDefinition()
             }
         }
         #endregion
+
+        #region Symbol - Static
+        [Export(typeof(EditorFormatDefinition))]
+        [ClassificationType(ClassificationTypeNames = ClassificationTypeNames.StaticSymbol)]
+        [Name(ClassificationTypeNames.StaticSymbol)]
+        [Order(After = PredefinedClassificationTypeNames.Identifier)]
+        [UserVisible(true)]
+        [ExcludeFromCodeCoverage]
+        private class SymbolStaticFormatDefinition : ClassificationFormatDefinition
+        {
+            private SymbolStaticFormatDefinition()
+            {
+                this.DisplayName = EditorFeaturesResources.Symbol_Static;
+                this.IsBold = true;
```

Comment:
❗️ Was not expecting any style changes in this pull request.

Reply:
Good point. Choosing style changes can come later if warranted.